providers/common/kdfs/tls1_prf.c

```diff
@@ -78,10 +79,18 @@ static int tls1_prf_alg(const EVP_MD *md, const EVP_MD *sha1,
 /* TLS KDF kdf context structure */
 typedef struct {
     void *provctx;
+
     /* Digest to use for PRF */
-    PROV_DIGEST digest;
+    char *digest;
     /* Second digest for the MD5/SHA-1 combined PRF */
-    PROV_DIGEST sha1;
+    const char *sha1;
```

Comment:
Why?  This one is constant?

Resolving early is a performance gain.

Reply:
Good question, it can be hard coded in `tls1_prf_alg`...